identity/identity.go

```diff
@@ -54,7 +56,12 @@ func NewIdentity(name string, email string) *Identity {
 	}
 }
 
-func NewIdentityFull(name string, email string, login string, avatarUrl string) *Identity {
```

Comment:
I think a key change for the identities would be to have a `KeyringAtTime(time lamport.Time) openpgp.EntityList` in `identity.Interface`. This could be used later with gogit's `Commit.Verify` function to let them figure out the correct key from the possible candidate valid at that time.

Reply:
I did introduce a lamport clock for identity versions, but for the regular commits we don't have any lamport time. This could be used only when checking the Identity/Version commits.